### DISCUSSION TITLE
Got 'invalid copilot token 403' even if I have copilot subscription

### DISCUSSION THREAD

**Discussion by labuladong** [community/community#115490] [Copilot Conversations]
### Select Topic Area

Bug

### Body

My copilot subscription is not overdue yet:

<img width="755" alt="image" src="https://github.com/community/community/assets/37220920/b7fe3dc9-ba3b-4783-b2e9-0a3d4efb8c92">

But vscode got `invalid copilot token 403` error when use Copilot:

<img width="1185" alt="image" src="https://github.com/community/community/assets/37220920/d095c887-5790-440d-a6ed-d8676066d7be">

It also fails in JetBrains IDE too:

<img width="440" alt="image" src="https://github.com/community/community/assets/37220920/b2bb1c8b-5f55-429a-8774-f7f93a685b69">

I've given feedback to [GitHub support](https://support.github.com/request) but no response.

**Comment by AperturePlus** [comment]
same :(

**Reply by gaopinghuang0** [reply]
same :(

**Comment by elbekgiyozov** [comment]
If you're getting an "invalid Copilot token 403" error despite having a subscription, ensure your token isn't expired, has the right permissions, and your subscription is active. If issues persist, contact GitHub Copilot support for assistance.

**Reply by labuladong** [reply]
Thanks for your reply.
1. I've re-login many times, but it still failed.
2. My subscription will be overdue on 4/6, so I believe it's active now.
3. I created a ticket at GitHub Copilot support yesterday, but no response yet.

**Reply by gaopinghuang0** [reply]
I also created a ticket several days ago and got a response that guides me to get a refund. However, my expectation is to keep using Copilot, not a refund :(

**Comment by davecheney** [comment]
I am sorry, we cannot assist you with this issue via this forum. Please contact support https://support.github.com/contact
